package/rebuild-todo

```diff
@@ -219,6 +224,22 @@ read <&1
 
 pkgctl repo clone "${packages[@]}"
 
+if ((IMPORT_KEYS)); then
+	echo -e "Importing PGP keys...\n"
+	# Only add paths that actually have key(s) to import, ignore errors and don't exit on paths that don't
+	key_paths=$(find "${packages[@]/%//keys/pgp}" -type f 2>/dev/null || true)
+
+	if [[ -n "$key_paths" ]]; then
+		if [[ -z $OFFLOAD ]]; then
+			cat $key_paths | gpg --import
+		else
+			cat $key_paths | ssh build.archlinux.org gpg --import
```

Comment:
Please properly quote these paths so that also weird paths (with spaces etc.) will work.
Then you'll most likely also need array expansion like this: `"${key_paths[@]}"` 😊

Reply:
Those where specifically unquoted to preserve word splitting between each paths. But indeed might work with `"${key_paths[@]}"` :)

Reply:
`key_paths` has been turned into an array and those commands are now using `"${key_paths[@]}"`